catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/CachingFederationStrategy.java

```diff
@@ -269,41 +267,35 @@ private QueryResponse sourceFederate(List<Source> sources, final QueryRequest qu
         // Do NOT call source.isAvailable() when checking sources
         for (final Source source : sources) {
             if (source != null) {
-                if (!futuresContainsSource(source, futures)) {
```

Comment:
The diff shows a little odd but the only thing changed in this method is the removal of this if statement.